### ISSUE TITLE
[A11Y] In the contributors panel, there are same level headings with the same text label in a row

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9358]
## Issue Description
// What is the nature of the issue? How does it impact users?

In the contributors panel of the new submission process, there are two headings level 2 that present the very same text label: Contributors

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Create a new submission
2. Go to step 3 of the process and you will encounter contributors

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4 

## Stack used 

- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Manual: Keyboard

## Action
// Describe action that can be taken to remediate the issue. 

Review the headings to adjust the appropriate Heading level and the text label. Maybe the second heading should become a heading level 3 or change to "List of Contributors"

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/a90dcb15-b1c7-43fa-af0c-19ed9a66d708)
